### PR TITLE
Add a demo presentation showcasing the features of pdfpcnotes

### PR DESCRIPTION
**Pull request by abooij**
We can add more demo code to this as features are added.

This is good for testing (what's supposed to work) and documentation.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -30,3 +30,7 @@
 *.vrb
 *.xdy
 *.tdo
+*.fls
+demo/test.pdf
+demo/test.pdfpc
+auto/
```
Comment (cebe) [inline .gitignore]: what is `auto/` ?

File: demo/test.tex [hunk 2]
```diff
@@ -0,0 +1,57 @@
+\documentclass{beamer}
+
+% The presentation is supposed to last 30 minutes, starting at 1PM.
+% pdfpc will give a warning for the last 5 minutes of the
+% presentation.
+\usepackage[duration=30,starttime=13:00,lastminutes=5]{../pdfpcnotes}
```
Comment (cebe) [inline demo/test.tex]: I'd prefer to use `pdfpcnotes` here without the path and use TEXINPUTS environment variable in the makefile like here:  https://github.com/cebe/pdfpc-latex-notes/blob/92a6dca9da1557c9305f41885556474b51ed46c4/tests/Makefile#L4

File: .gitignore (hunk 1 above)
Comment (abooij) [reply .gitignore]: I'm not sure; for some reason latex compilers seem to generate this directory for me sometimes.

File: demo/test.tex (hunk 2 above)
Comment (abooij) [reply demo/test.tex]: I guess essentially my proposal is just another item in your new `tests/` directory. In particular it tests overlays and `\againframe`. So perhaps this PR should be reorganised. Feel free to pick out what you like.

Comment (cebe) [reply demo/test.tex]: I like to have a demo presentation which serves also as part of the documentation. noone would look at tests when trying to figure out how something works.

**Comment by cebe** [comment]
Thank you!

**Pull request merged**